src/services/namingService.test.ts

```diff
@@ -86,4 +86,60 @@ describe("Naming Service", () => {
     const actual = AzureNamingService.getNormalizedRegionName(expected);
     expect(actual).toEqual(expected);
   });
+
+  it("deployment name is generated correctly", () => {
```

Comment:
I would put these 2 new tests in the deployment class test

Reply:
They are related to the naming service - so going to keep these in here for now.